lapis-docs/src/content/docs/concepts/advanced-query.mdx

```diff
@@ -0,0 +1,138 @@
+seen a superset of variant queries, in which not only variant information can be queried but also metadata information.
+
+Advanced queries can be used to filter sequences and metadata and can be passed to the server through the query parameter
+`advancedQuery`. Don't forget to encode/escape the query correctly (in JavaScript, this can be done with the
+[encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)
+function)!
+
+The formal specification of the query language is available
+[here](https://github.com/GenSpectrum/LAPIS/blob/main/lapis/src/main/antlr/org/genspectrum/lapis/model/advancedqueryparser/AdvancedQuery.g4)
+as an ANTLR v4 grammar.
+In following, we provide an informal description and examples.
+The respective
+[unit test](https://github.com/GenSpectrum/LAPIS/blob/main/lapis/src/test/kotlin/org/genspectrum/lapis/model/AdvancedQueryFacadeTest.kt)
+provides a full list of possible atomic queries.
+
+The query language understands Boolean logic. Expressions can be connected with `&` (and), `|` (or) and `!` (not).
+Both `&` and `AND` are recognized as `and`, `|` and `OR` are recognized as `or`, and `!` and `NOT` are recognized as `not`.
+Parentheses `(` and `)` can be used to define the order of the operations. Further, there is a special syntax to match
+sequences for which at least or exactly `n` out of a list of expressions are fulfilled. For variants we additionally add
+the `MAYBE` operator to query [ambiguous nucleotide symbols](../concepts/ambiguous-symbols).
+
+For metadata fields we also add custom queries to allow for regex substring search, is null queries (to search for fields
+which are empty or `null`) as well as range queries for dates and numbers.
+
+## Examples
+
+Get the sequences with the nucleotide mutation 300G, without a deletion at position 400 and either the AA change S:123T
+or the AA change S:234A:
+
+```
+300G & !400- & (S:123T | S:234A)
+```
+
+This can also be written as
+
+```
+300G AND NOT 400- AND (S:123T OR S:234A)
+```
+
+Get the sequences with at least 3 out of five mutations/deletions:
+
+```
+[3-of: 123A, 234T, S:345-, ORF1a:456K, ORF7A:100-]
+```
+
+Get the sequences that fulfill exactly 2 out of 4 conditions:
+
+```
+[exactly-2-of: 123A & 234T, !234T, S:345- | S:346-, [2-of: 222T, 333G, 444A, 555C]]
+```
+
+Get sequences that might have the mutation 123W (this includes ambiguous symbols and `N`):
+
+```
+MAYBE(123W)
+```
+
+We can also add metadata queries to the advanced query.
+
+For example, to get all sequences from the USA that do not have cows as a host and that also have the mutation 300G:
+
+```
+NOT host='bos taurus' AND 300G AND country=USA
+```
+
+Note that if the metadata field does not only contain letters and numbers (e.g. it contains a space or a regex),
+it must be enclosed in single quotes.
+
+!! When passing single quotes in curl they must be escaped correctly, for example the statement:
```

Comment:
IMO this formatting looks a little better:
```suggestion
:::note
When passing single quotes in curl they must be escaped correctly, for example the statement:
```

(and then closing the note with `:::` after the curl example)